tests/test_sign_cmd.py

```diff
@@ -55,13 +51,13 @@ def test_sign_tx_change_protocol_v1(cmd, public_key_bytes):
         )
         for x in range(5)
     ]
-    change_indices = [fake.pyint(0, 5) for x in range(5)]
+    change_indices = list(set([fake.pyint(0, 4) for x in range(5)]))
```

Comment:
This is because the range was [0, 5] meaning we may add an output index not on the tx (the 5 when indexes go up to 4).

I also changed it to be a `list(set(...))` to avoid repeating values. This could be done with `fake.unique.pyint` but it would just generate a `[0, 1, 2, 3, 4]` array (in a random order). My approach actually changes the length of the array, having always at least 1 change and up to 5.